src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/HandlerResolver.cpp

```diff
@@ -56,7 +56,7 @@ HandlerResolver::LoadRequestHandlerAssembly(const IHttpApplication &pApplication
     {
         if (pConfiguration.QueryHostingModel() == APP_HOSTING_MODEL::HOSTING_IN_PROCESS)
         {
-            errorContext.generalErrorType = "ANCM In-Process Handler Load Failure";
+            errorContext.generalErrorType = "ASP.NET Core Module In-Process Handler Load Failure";
```

Comment:
```suggestion
            errorContext.generalErrorType = "ASP.NET Core IIS hosting failure (in-process)";
```

The kind of hosting is more a modifier here :). Also, I don't know that I like Title Case For Error Messages.